### PR TITLE
Comment Details Top Padding

### LINKED ISSUE #4151: CommentsViewController: iPad Lacks Top Padding
ios simulator screen shot aug 13 2015 5 19 20 pm

### PR DESCRIPTION
**Pull request by jleandroperez**
#### Details:
In this PR we're adding a top padding to `CommentViewController `, on iPad devices. Towards the #4128 goal, we're also normalizing the topPadding -within the commentsView- to be iOS's default (35pt instead of 40pt)

--

#### Steps: Comments Top Padding
1. Launch WPiOS and log into your account
2. Tap over `My Sites`
3. Open any site
4. Tap over Comments

Expected: iPhone devices should have no top padding, while iPads should get 35pt top

--

#### Steps: Comment Details Top Padding
1. Launch WPiOS and log into your account
2. Tap over `My Sites`
3. Open any site
4. Tap over Comments and open any comment

Expected: iPhone devices should have no top padding, while iPads should get 35pt top

--

#### Steps:
1. Launch WPiOS and log into your account
2. Tap over `My Sites`

Please, test all of the rows, since the handling code has been normalized / cleaned up.

Closes #4151

Needs review: @aerych 
Thank you sir!

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Blog/BlogDetailsViewController.m [hunk 1]
```diff
@@ -312,13 +312,13 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
...
-    }
-
-    Class controllerClass;
-    if ([self isGeneralSection:indexPath.section]) {
+    if ([self isConfigurationSection:indexPath.section]) {
+        switch (indexPath.row) {
+            case BlogDetailsRowEditSite:
+                [self showSettingsForBlog:self.blog];
+                break;
+        }
```
Comment (aerych) [inline WordPress/Classes/ViewRelated/Blog/BlogDetailsViewController.m]: I guess it makes sense for consistency with the rest of the conditionals to have a nested switch with a single case, but I'm a little worried that this change doesn't handle a situation where `indexPath.row != BlogDetailsRowEditSite`. Maybe a default clause with an assert / log to at least flag that potential case?

Comment (aerych) [inline WordPress/Classes/ViewRelated/Blog/BlogDetailsViewController.m]: Also, if I'm being overly paranoid you should say so :)

**Comment by aerych** [comment]
One minor comment, take it or leave it :)  
Looks good!  :shipit:

File: WordPress/Classes/ViewRelated/Blog/BlogDetailsViewController.m (hunk 1 above)
Comment (jleandroperez) [inline WordPress/Classes/ViewRelated/Blog/BlogDetailsViewController.m]: No, you're actually right Eric. In Swift we'd get a nice warning about 'non extensive switch', in ObjC... i'm pretty sure nothing would happen. 

Adding assertions, thank you sir!

**Comment by jleandroperez** [comment]
Thank you @aerych!!

**Pull request merged**